### DISCUSSION TITLE
Can I use Alice with normal pip and vscode ?

### DISCUSSION THREAD

**Discussion by ghost** [TheAliceProject/alice3#315] [Q&A] [answered]
Can I use Alice with normal pip and vscode ?
I dont want to install an external IDE to make my things, is that possible ?

**Comment by dabeshouse** [comment, marked as answer]
Are you looking at Alice 2 or Alice 3?

What sort of things do you want to make?
Alice projects, but not inside the Alice IDE?
Extensions to Alice projects that go beyond what Alice offers?
  For both of these, the Alice 3 NetBeans plugin will allow you to build and extend worlds in Java.
Improvements or extensions to the Alice environment?
  If you have such things, we welcome PRs on GitHub.
